packages/cli/test/integration/passwordReset.api.test.ts

```diff
@@ -47,27 +47,34 @@ afterAll(async () => {
 	await testDb.terminate(testDbName);
 });
 
-test('POST /forgot-password should send password reset email', async () => {
-	const owner = await testDb.createUser({ globalRole: globalOwnerRole });
-
-	const authlessAgent = utils.createAgent(app);
-	const member = await testDb.createUser({ email: 'test@test.com', globalRole: globalMemberRole });
-
-	await utils.configureSmtp();
-
-	await Promise.all(
-		[{ email: owner.email }, { email: member.email.toUpperCase() }].map(async (payload) => {
-			const response = await authlessAgent.post('/forgot-password').send(payload);
-
-			expect(response.statusCode).toBe(200);
-			expect(response.body).toEqual({});
-
-			const user = await Db.collections.User.findOneOrFail({ email: payload.email });
-			expect(user.resetPasswordToken).toBeDefined();
-			expect(user.resetPasswordTokenExpiration).toBeGreaterThan(Math.ceil(Date.now() / 1000));
-		}),
-	);
-});
+test(
+	'POST /forgot-password should send password reset email',
+	async () => {
+		const owner = await testDb.createUser({ globalRole: globalOwnerRole });
+
+		const authlessAgent = utils.createAgent(app);
+		const member = await testDb.createUser({
+			email: 'test@test.com',
+			globalRole: globalMemberRole,
+		});
+
+		await utils.configureSmtp();
+
+		await Promise.all(
+			[{ email: owner.email }, { email: member.email.toUpperCase() }].map(async (payload) => {
+				const response = await authlessAgent.post('/forgot-password').send(payload);
+
+				expect(response.statusCode).toBe(200);
+				expect(response.body).toEqual({});
+
+				const user = await Db.collections.User!.findOneOrFail({ email: payload.email });
```

Comment:
shouldn't this be that?
```suggestion
				const user = await Db.collections.User.findOneOrFail({ email: payload.email });
```